api/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -97,4 +97,12 @@ message LocalRateLimit {
   //
   //  The filter supports a range of 0 - 10 inclusively for stage numbers.
   uint32 stage = 9 [(validate.rules).uint32 = {lte: 10}];
+
+  // Specifies the scope of the rate limiter's token bucket.
+  // If set to false, the token bucket is shared across all worker threads,
```

Comment:
optional: it would be nice to make it clear here, as you did in docs, that false is the default.
(it's generally clear if you know proto but I'm not sure if everyone reading Envoy APIs knows proto defaults)

Reply:
Sounds good! I'll make it explicit as to what would be the default value if unspecified.